modules/net-lb-int/main.tf

```diff
@@ -25,24 +25,26 @@ locals {
   )
 }
 
-resource "google_compute_forwarding_rule" "default" {
+resource "google_compute_forwarding_rule" "forwarding_rules" {
+  for_each    = var.forwarding_rules_config
   provider    = google-beta
   project     = var.project_id
   region      = var.region
-  name        = var.name
-  description = var.description
-  ip_address  = var.address
-  ip_protocol = var.protocol
+  name        = "${var.name}-${each.key}"
```

Comment:
I would do

```hcl
name = (
  each.key == "" ? var.name : "${var.name}-${each.key}"
)
```

The common use case that probably covers 99.9% of usage (as we never had support for multiple forwarding rules till now) is to use a single rule, and in that case the name should match names used for all other resources like backend services, health check, etc.